Felienne/hedy
# [Feature] Adds adventures overview to admin page

**Description**
With the introduction of custom adventures for teachers, they are able to create and use their own adventures within their classes. In this PR we add a feature enabling the admin to get an overview of all these adventures.

**Fixes #1965*

**How to test**
Make sure your username is set as `is_admin` in the environment variables. Navigate to `/admin` and notice the new button called `Adventures overview`. Click it and verify that all created adventures are shown in a table. The admin should be able to view all adventures, whereas other users can not.

**Checklist**
Done? Check if you have it all in place using this list*
  

* If you're unsure about any of these, don't hesitate to ask. We're here to help!
